modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1180 @@
+
+To fix this bug, in {productname} 6.4, a blocker was added to the `editor.insertContent` api and `mceInsertContent` command, which stops `contenteditable="false"` root elements from being edited.
+
+In {productname} 6.4, inserting contents inside a `contenteditable="false"` element while the selection is inside another element, is no longer possible.
+
+=== Closing a dialog would scroll down the document in Safari
+// TINY-9148
+
+While using previous versions of {productname}, `clicking` away to dismiss the *Search* and *Replace* dialog within Safari would scroll the page to the bottom of the editor. As a result the user would have to manually scroll back to the original `focus` area each time the steps above were executed.
+
+To correct this issue, a new Css.set was implemented to keep the `focus` on the DOM container in the initial scroll document position. 
+
+In {productname} 6.4, when the user `clicks` away to dismiss the *Search* and *Replace* dialog within Safari, the focus now returns to the editor and the scroll does not change position within the editor.
+
+=== Inline headers would not work in some situations when the editor was moved too far right horizontally
+// TINY-8977
+
+// Cause
+// Consequence
+// Fix
+// Result
+
+=== Quick toolbars were incorrectly rendered during the dragging of contenteditable="false" elements
+// TINY-9305
+
+The quickbar was displaying incorrectly during a drag and drop `action` for contenteditable="false" elements. As a consequence, during the movement of a contenteditable="false" element, the selection was updated, which triggered a *NodeChange*. This selection update would cause an absence of the caret location during a dragging or dropping event.
+
+To fix this bug, {productname} 6.4 added a new check to identify when the user is dragging a contenteditable="false" element, which blocks the quickbar from rendering.
+
+As a result, the quickbar no longer opens in {productname} v6.4+, when the user is dragging a contenteditable="false" elements.
+
+=== Selection of images, hrs, tables or noneditable elements was possible if they were within a noneditable element
+// TINY-9473
+
+In previous versions of {productname}, `contenteditable="false"` elements that were nested within another such as a element set with the `contenteditable="false"` attribute, would avoid the `contenteditable="false"` guidelines set. As a consequence, selection of images, hrs, tables or `contenteditable="false"` elements was possible if they were nested within a `contenteditable="false"` element.
+
+Improvements were made to the `SelectionOverride` to now check if the DOM parent node is either set as a `contenteditable="true"` element or as a `contenteditable="false"` element.
+
+In {productname} 6.4, elements with `contenteditable="false"` attributes can no longer be selected when they are children of another `contenteditable="false"` attribute, such as:
+
+.Example of `contenteditable="false"` elements
+[source,HTML]
+----
```

Comment:
This is a bad example since it doesn't have a child noneditable element. Also, what is this example trying to convey?